compiler/src/org.graalvm.compiler.serviceprovider/src/org/graalvm/compiler/serviceprovider/GraalServices.java

```diff
@@ -71,6 +71,11 @@ private static int getJavaSpecificationVersion() {
      */
     public static final boolean Java8OrEarlier = JAVA_SPECIFICATION_VERSION <= 8;
 
+    /**
+     * Determines if the Java runtime is version 8 or earlier.
+     */
+    public static final boolean Java11OrLater = JAVA_SPECIFICATION_VERSION >= 11;
```

Comment:
Do you really need to know if you are on 11 or later?  I don't think SubstrateVM ever plans to (or can) run on 9 or 10 because of JVMCI changes; I think we are going straight to 11.

I am also not sure if I can make changes to the public API of org.graalvm.compiler.serviceprovider.GraalServices.  But I will check.

Reply:
This API was added in Java 11 and is not present in 9 or 10, so saying 9-or-later might be confusing to say the least.  That said, it doesn't actually matter too much to me.  I can change it in whatever manner you think is best.

Reply:
It turns out this is internal API, so adding `Java11OrLater` is fine.